.lintstagedrc

```diff
@@ -0,0 +1,3 @@
+{
+  "*.{js,ts}": ["prettier --write", "git add"]
```

Comment:
Wouldn’t it make sense to include json so that schemas are formatted as well?

Reply:
Wanted to start with the code first.  There's also tests that rely on specific JSON file contents so there's some effort there to get the exclusions correct.